tests/tests/tier0/bluechi-monitor-service/python/monitor.py

```diff
@@ -1,122 +1,56 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
Looking at the refactored code, the `python/monitor.py` wouldn't be necessary - everything could be done in the `test_bluechictl_monitor_xyz`.

Reply:
How? We need the FileFollower and all that stuff should run in the container itself

Reply:
Hmm... the FileFollower isn't a problem, but the BackgroundRunner is due to the `subprocess.Popen`. Lets keep it as it is for now.